roqoqo-qryd/src/api_devices.rs

```diff
@@ -34,6 +36,7 @@ impl QRydAPIDevice {
         match self {
             Self::QrydEmuSquareDevice(x) => x.qrydbackend(),
             Self::QrydEmuTriangularDevice(x) => x.qrydbackend(),
+            Self::TweezerDevice(x) => x.qrydbackend.clone(),
```

Comment:
How come this is called as an attribute here, instead of a function, for the sake of consistency?

Reply:
No real reason, I simply didn't understand the necessity of creating a new function if the attribute can simply be made public.